src/TiledArray/tensor/kernels.h

```diff
@@ -587,7 +587,7 @@ namespace TiledArray {
       Scalar result = identity;
       for(decltype(tensor1.range().volume()) i = 0ul; i < volume; i += stride) {
         Scalar temp = identity;
-        math::reduce_op(reduce_op, stride, temp,
+        math::reduce_op(reduce_op,join_op, stride, temp,
```

Comment:
Why is this call to reduce_op different from the one on line 519? Where is the identity argument? Watch the formatting.